### PR TITLE
ovirt: doc: Add TOC and oVirt logo to documentation

### PR DESCRIPTION
**Pull request by dougsland**
- Add Table of Contents to oVirt documentation.
- Add oVirt Logo
- Fix the order for bootstrap vm and ovirt-config.yaml

Signed-off-by: Douglas Schilling Landgraf <dougsland@redhat.com>

### CODE REVIEW

File: docs/user/ovirt/install_ipi.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+![oVirt Logo](images/oVirt-logo.png#center)
```
Comment (abhinavdahiya) [inline docs/user/ovirt/install_ipi.md:1]:
```suggestion
![oVirt Logo](./images/oVirt-logo.png#center)
```

**Comment by Gal-Zaidman** [comment]
Hi @dougsland,
First of all, love the fact that you took it on your self to edit this! great initiative,
what do you think about syncing with DOCs and adding all that is required to install via ipi?
meaning:
- env variables that are available
- install config params
- prerequisites (we know this is what will cause weird bugs to be reported)
- maybe a small Demo?
even in the form of a link to the official docs

**Comment by dougsland** [comment]
> Hi @dougsland,
> First of all, love the fact that you took it on your self to edit this! great initiative,
> what do you think about syncing with DOCs and adding all that is required to install via ipi?
> meaning:
> 
> * env variables that are available
> * install config params
> * prerequisites (we know this is what will cause weird bugs to be reported)
> * maybe a small Demo?
>   even in the form of a link to the official docs

Sure @Gal-Zaidman. Thanks for the suggestion. I I will add in my TODO list and create separate patches.

**Pull request merged**